docs/user/openstack/install_upi.md

```diff
@@ -102,7 +102,7 @@ This repository contains [Ansible playbooks][ansible-upi] to deploy OpenShift on
 They can be downloaded from Github with this script:
 
 ```sh
-RELEASE="release-4.6"; xargs -n 1 curl -O <<< "
+RELEASE="release-4.13"; xargs -n 1 curl -O <<< "
```

Comment:
Honestly, we could leave `release-4.6` here, we explain to change the name of the release... This would facilitate the backports to not touch the release name.

Reply:
I messed up with the cherry-picks, it's good now.

Reply:
+1 to Martin's point.

I wonder if we need this one https://github.com/openshift/installer/pull/7385/commits/b179135230a1f72c8f9b94138af36b2c3c213f87